src/block/block_ext.c

```diff
@@ -1003,12 +1011,18 @@ __block_append(
      * object in the skiplist, check for a simple extension, and otherwise append a new structure.
      */
     if ((ext = el->last) != NULL && ext->off + ext->size == off)
+        /* Extend the last object on the list. off is adjacent to the end of the last extent.*/
         ext->size += size;
     else {
+        /* Do the lazy update of el->last requested in __block_off_remove. */
```

Comment:
I think this comment is misplaced. `__block_off_srch_last` doesn't update the cached end-of-list.

Reply:
Done. I  deleted it.

You are right. This comment is misplaced. It should have been after th other else before the call to __block_off_srch_last.